src/recipes/eas-update.ts

```diff
@@ -1,86 +1,59 @@
-        'Detected eas.json, skipping EAS Build configuration.'
-      )
-    } else {
-      await toolbox.interactive.spawnSubprocess(
-        'EAS Build configuration',
-        'eas build:configure -p all'
-      )
-
-      toolbox.interactive.step('Created default EAS Build configuration.')
-    }
-
-    await toolbox.interactive.spawnSubprocess(
-      'EAS Update configuration',
-      'eas update:configure'
-    )
-
-    await toolbox.workflows.generate(join('eas', 'eas-update.ejf'), context)
+const execute = async (
+  toolbox: CycliToolbox,
+  context: ProjectContext
+): Promise<void> => {
+  toolbox.interactive.vspace()
+  toolbox.interactive.sectionHeader(
+    'Genereating EAS Update and Preview workflow'
+  )
 
-    toolbox.interactive.success('Created EAS Update and Preview workflow.')
+  toolbox.dependencies.add('expo', context)
 
-    toolbox.interactive.warning(
-      `Remember to create repository secret EXPO_TOKEN for EAS Update workflow to work properly. For more information check ${REPOSITORY_SECRETS_HELP_URL}`
+  if (toolbox.filesystem.exists('eas.json')) {
+    toolbox.interactive.step(
+      'Detected eas.json, skipping EAS Build configuration.'
     )
-    toolbox.furtherActions.push(
-      `Create EXPO_TOKEN repository secret. More info at ${REPOSITORY_SECRETS_HELP_URL}`
+  } else {
+    await toolbox.interactive.spawnSubprocess(
+      'EAS Build configuration',
+      'eas build:configure -p all'
     )
 
-    return `--${FLAG}`
+    toolbox.interactive.step('Created default EAS Build configuration.')
   }
 
-const run = async (
-  toolbox: CycliToolbox,
-  context: ProjectContext
-): Promise<RunResult> => {
-  let runRecipe = false
+  await toolbox.interactive.spawnSubprocess(
+    'EAS Update configuration',
+    'eas update:configure'
+  )
 
-  if (toolbox.options.isRecipeSelected(FLAG)) {
-    runRecipe = true
-  } else if (!toolbox.options.isPreset()) {
-    runRecipe = await toolbox.interactive.confirm(
-      'Do you want to generate preview with EAS Update on every PR? (Expo projects only)',
-      { type: 'normal' }
-    )
-  }
-
-  if (runRecipe) {
-    // EAS Update recipes is currently supported only for Expo projects
-    if (!toolbox.projectConfig.isExpo()) {
-      throw Error('EAS Update workflow is supported only for Expo projects.')
-    }
+  await toolbox.workflows.generate(join('eas', 'eas-update.ejf'), context)
 
-    context.selectedOptions.push(FLAG)
-
-    return execute()
-  }
+  toolbox.interactive.success('Created EAS Update and Preview workflow.')
 
-  return null
+  toolbox.interactive.warning(
+    `Remember to create repository secret EXPO_TOKEN for EAS Update workflow to work properly. For more information check ${REPOSITORY_SECRETS_HELP_URL}`
+  )
+  toolbox.furtherActions.push(
+    `Create EXPO_TOKEN repository secret. More info at ${REPOSITORY_SECRETS_HELP_URL}`
+  )
 }
 
 export const recipe: CycliRecipe = {
   meta: {
+    name: 'EAS Update and Preview',
```

Comment:
Same here and probably in docs etc

Reply:
Changed to "Preview with EAS Update" here and in readme 9c1def7afe372e509ae91a2ab9f59e75535a1bd8